### PR TITLE
making response ops for account email change consistent

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/yk676X1g/1225-change-rops-resend-email-to-use-the-new-template

### CODE REVIEW

File: ras_party/controllers/account_controller.py [hunk 1]
```diff
@@ -58,6 +58,7 @@
 
 NO_RESPONDENT_FOR_PARTY_ID = "There is no respondent with that party ID"
 EMAIL_VERIFICATION_SENT = "A new verification email has been sent"
+ACCOUNT_EMAIL_CHANGE_VERIFICATION_SENT = "A new account email change verification email has been sent"
```
Comment (insacuri) [inline ras_party/controllers/account_controller.py]: This reads kinda weird.... maybe
`A verification email has been sent to change the email address on the account` or something like that?  My suggestion kinda sucks so don't use that!  Can you ask for possibly better wording for it from someone else?

File: ras_party/views/account_view.py [hunk 2]
```diff
@@ -76,6 +76,12 @@ def resend_verification_email(party_uuid):
     return make_response(jsonify(response), 200)
 
 
+@account_view.route("/resend-account-email-change-notification/<party_uuid>", methods=["POST"])
+def resend_account_email_change_verification(party_uuid):
+    response = account_controller.resend_account_email_change_verification_email_by_uuid(party_uuid)
+    return make_response(jsonify(response), 200)
```
Comment (insacuri) [inline ras_party/views/account_view.py:82]: This can just be `jsonify(response)`

**Review by insacuri — CHANGES_REQUESTED** [review]
2 small changes but overall, looks good

File: ras_party/views/account_view.py (hunk 2 above)
Comment (creativeappdevamit) [reply ras_party/views/account_view.py:82]: for consistency we should add the http code I believe

File: ras_party/controllers/account_controller.py (hunk 1 above)
Comment (creativeappdevamit) [reply ras_party/controllers/account_controller.py]: 👍

**Review by insacuri — APPROVED** [review]

**Pull request merged**